e2e/nextjs/app-router/app/yarn.lock

```diff
@@ -0,0 +1,671 @@
+# This file is generated by running "yarn install" inside your project.
+# Manual changes might be lost - proceed with caution!
+
+__metadata:
```

Comment:
Should this have its own `yarn.lock` file?

Reply:
yeah its a fully isolated app